src/tests/class_tests/openms/source/LPWrapper_test.cpp

```diff
@@ -331,55 +330,48 @@ END_SECTION
 
 START_SECTION((void writeProblem(const String &filename, const WriteFormat format) const ))
 {
-  if(lp.getSolver() == LPWrapper::SOLVER_GLPK)
```

Comment:
I would leave the `if (getSolver() ==`) else if (getsolver ==)`, because it saves you from knowing anything about the implementation details and avoids the `#ifdef` ugliness in the test.